src/ExpandObjects/epfilter.f90

```diff
@@ -31850,6 +31855,7 @@ SUBROUTINE CreateDOAS
          base + doasNameOff,' DX Coil EIR-FF')
       CALL AddToObjFld('Part Load Fraction Correlation Curve Name',   &
          base + doasNameOff,' DX Coil PLF')
+      CALL AddToObjStr('Minimum Outdoor Dry-Bulb Temperature for Compressor Operation {C}','')
```

Comment:
@mjwitte These all look right to me.  I'll build and run expand objects on a test file to verify.